extensions/amp-bind/0.1/bind-impl.js

```diff
@@ -100,10 +101,15 @@ export class Bind {
     this.resources_ = resourcesForDoc(ampdoc);
 
     /**
-     * True if a digest is triggered before scan for bindings completes.
-     * @private {boolean}
+     * @const @private {!Array<Promise>}
      */
-    this.digestQueuedAfterScan_ = false;
+    this.mutationPromises_ = [];
```

Comment:
Is this array ever cleaned up in production? If it's only used for testing, you can document it as such and add `if (getMode().test)` guards as I suggested elsewhere.